eos-prune-printers

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env sh
+
+# eos-prune-printers: Purge all installed printers and corresponding drivers
+# Copyright (C) 2021 Endless OS Foundation LLC
+# Licensed under the GPLv2
+
+# Exit if the stamp file exists. The service file checks for this, but also
+# guard against an accidental execution.
+stamp_file=/var/eos-prune-printers
+[ -e $stamp_file ] && exit 0
+
+# Purge and cancel all print jobs
+cancel -xa
+
+# Delete all printers with CUPS tool
+for printer in `lpstat -p | awk '{print $2}'`
+do
+	echo Deleting $printer
+	lpadmin -x $printer
```

Comment:
I think you should quote the variable since we don't know what's in it (spaces, shell special characters, etc.).

Reply:
Sure